pytorch_lightning/strategies/strategy.py

```diff
@@ -442,6 +442,9 @@ def teardown(self) -> None:
         optimizers_to_device(self.optimizers, torch.device("cpu"))
         self.precision_plugin.teardown()
 
+        if self.launcher:
```

Comment:
careful, above in the docstring: "teardown the training process"
Bad wording, we don't mean the process as in "subprocess" here, rather the "training procedure/run"